recOrder/scripts/calibrate.py

```diff
@@ -0,0 +1,30 @@
+# This script can be modified to debug and test calibrations
```

Comment:
Is this script best suited in the `recOrder` package (thus released with source code on PyPI)?

Reply:
Fair point. A possible counterargument is that we release unit tests (although they run automatically whereas these have much stronger dependencies). 

What do you think about moving this script (and others I generated today) to `compmicro-sandbox`?

Reply:
Scripts to test calibration and acquisition can come be helpful when our collaborators run into bugs involving optical components and acquisition. Let's keep these scripts in `recOrder` repo. I'm fine with sharing the scripts with PyPI package. If we encounter a problem packaging these scripts, we can [exclude](https://stackoverflow.com/questions/45111877/exclude-a-folder-from-a-python-package-for-uploading-a-package-to-pypi) the scripts folder.

Reply:
Roger that...let's include this script.